dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/stream/StreamUtils.java

```diff
@@ -88,27 +101,26 @@ public static void convertAttachment(DefaultHttp2Headers headers,
         if (attachments == null) {
             return;
         }
-
+        Map<String, String> needConvertKey = new HashMap<>();
         for (Map.Entry<String, Object> entry : attachments.entrySet()) {
-            final String key = entry.getKey().toLowerCase(Locale.ROOT);
-            if (Http2Headers.PseudoHeaderName.isPseudoHeader(key)) {
-                continue;
+            String key = lruHeaderMap.get(entry.getKey());
+            if (key == null) {
+                final String lowerCaseKey = entry.getKey().toLowerCase(Locale.ROOT);
+                lruHeaderMap.put(entry.getKey(), lowerCaseKey);
```

Comment:
concrruent issue

Reply:
The concurrency problem here is ignored and does not affect the final result